java/squeek/appleskin/client/HUDOverlayHandler.java

```diff
@@ -74,7 +74,9 @@ public void onRender(RenderGameOverlayEvent.Post event)
 
 		Minecraft mc = Minecraft.getInstance();
 		PlayerEntity player = mc.player;
-		ItemStack heldItem = player.getHeldItemMainhand();
+		ItemStack heldItem;
+		if(!FoodHelper.isFood(heldItem = player.getHeldItemMainhand()))
```

Comment:
```suggestion
		ItemStack heldItem = player.getHeldItemMainhand();
		if(!FoodHelper.isFood(heldItem))
```

Not a fan of assignment in conditionals